preprocessing.py

```diff
@@ -56,10 +69,26 @@ def deserializeIndex(filePath):
         all_terms_in_dict = pickle.loads(s)
         return all_terms_in_dict
 
+def sliding_window(size_of_window, docid, dictionary_of_all_terms):
+    with open(docid, 'r') as my_file:
+        data = my_file.read().split()
+    k = len(data) // size_of_window
+    dimension = len(dictionary_of_all_terms)
+    result = []
+    for l in range(0, k):
+        current_vector = sparse.coo_matrix((dimension, 1))
+        for e in range(0, size_of_window):
+            current_vector += sparse_basis_vector_creation(dictionary_of_all_terms[data[e]], dimension)
```

Comment:
for examle
If you use lil_matrix you can write code like this:

m = sp.lil_matrix((10, 1))
m[(0, 0)] = 1
m[(0, 0)]

i.e. you can use indexing and rewrite line to:
current_vector[(dictionary_of_all_terms[data[e]], 0)] += 1 
as I understand